oshdb-api/src/main/java/org/heigit/bigspatialdata/oshdb/api/mapreducer/backend/MapReducerIgniteLocalPeek.java

```diff
@@ -191,10 +190,7 @@ public boolean isActive() {
       final int bufferSize = 102400 * ForkJoinPool.commonPool().getParallelism();
 
       CellKeysIterator(Iterable<CellIdRange> cellIdRanges) {
-        this.cellIds = StreamSupport.stream(
-            Spliterators.spliteratorUnknownSize(cellIdRanges.iterator(), 0),
-            true
```

Comment:
:thinking: `Streams.stream(…)` in the new version doesn't set the parallel flag. :see_no_evil: